core/datatype.h

```diff
@@ -144,8 +145,9 @@ namespace MR
       static constexpr uint8_t     UInt16        = 0x03U;
       static constexpr uint8_t     UInt32        = 0x04U;
       static constexpr uint8_t     UInt64        = 0x05U;
-      static constexpr uint8_t     Float32       = 0x06U;
-      static constexpr uint8_t     Float64       = 0x07U;
+      static constexpr uint8_t     Float16       = 0x06U;
+      static constexpr uint8_t     Float32       = 0x07U;
+      static constexpr uint8_t     Float64       = 0x08U;
```

Comment:
I don't think there's any context in which these numerical identifiers are important, the datatype is always translated to string and back again; so putting them in this order shouldn't be a problem?